kafka/source.go

```diff
@@ -124,44 +124,15 @@ func (m MergedMetadata) Merge(v streams.Metadata) streams.Metadata {
 	return merged
 }
 
-// Metadata represents the position in the stream of a message.
-type Metadata struct {
+// PartitionOffsets represents the position in the stream of a message.
+type PartitionOffsets struct {
```

Comment:
`PartitionOffsets` represents a single offset of a single partition. Let's keep the name singular.